src/apps/Like/components/LikesViewComponent.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {Link} from "react-router-dom";
+
+const image = "https://emmanuelbeja.github.io/ah-jumaji-ui/assets/img/default-avatar1.jpg";
```

Comment:
Put this in config file with a suitable name

Reply:
Done. I have implemented it.